src/ui/buttons/buttonDesigns/buttonDesignComponent.jsx

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+/** @jsx node */
+import { node, Fragment, type ChildType } from 'jsx-pragmatic/src';
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+
+import { CLASS } from '../../../constants';
+
+import { type LabelOptions } from './types';
+
+export function ButtonDesignComponent({ designLabelText } : LabelOptions) : ChildType {
+    return (
+        <Fragment>
+            <div class={ 'personalized-label-container' } data-animation-experiment="Varied_Button_Design"> <span>{designLabelText}</span></div>
+            <style innerHTML={ `
```

Comment:
This style tag should be optional( since the styles can be created at keyframes creation time ), and would be helpful to receive it as a property in case we need them. 
Take a look at this animation, where I was able to remove the styles from here, and I add them within the keyframes creation script.
![Screen Shot 2021-11-15 at 12 55 54 PM](https://user-images.githubusercontent.com/7244087/141831038-bf9ed014-9700-42dd-953f-88c2c7705b09.png)

Reply:
They definitely can be moved, but I'm not sure I understand what the advantage of doing this would be?